config.json

```diff
@@ -3,9 +3,14 @@
   "language": "PureScript",
   "repository": "https://github.com/exercism/xpurescript",
   "active": false,
-  "test_pattern": "TODO",
-  "problems": [
-
+  "exercises": [
+    {
+      "slug": "hello-world",
+      "difficulty": 1,
+      "topics": [
+        "maybe"
```

Comment:
At first I thought this was a placeholder, until I realized that PureScript has a `maybe` :)